src/cpp/hu.bme.mit.inf.modes3.ESP32-sensor/railsensor_3/ESP32-sensor/lib/InfraSensor/InfraSensor.hpp

```diff
@@ -1,103 +1,49 @@
 #ifndef SENSOR_PRELLTIME
-    #define SENSOR_PRELLTIME 0.01
+#define SENSOR_PRELLTIME 0.01
 #endif
 
 #ifndef INFRASENSOR_HPP
 #define INFRASENSOR_HPP
 
-double SecTime(){
-    double ret=micros()/1000000.0;
-    return ret;
+double SecTime()
+{
+  double ret = micros() / 1000000.0;
+  return ret;
```

Comment:
You could return the computeed value directly, there is no need for a 'noname' intermediate variable.